src/main/kotlin/no/nav/familie/ef/sak/barn/BarnService.kt

```diff
@@ -87,30 +87,30 @@ class BarnService(
         ustrukturertDokumentasjonType: UstrukturertDokumentasjonType,
         grunnlagsdataBarn: List<BarnMedIdent>,
         barnSomSkalFødes: List<BarnSomSkalFødes>,
-        vilkårsbehandleNyeBarn: VilkårsbehandleNyeBarn
+        vilkårsbehandleNyeBarn: VilkårsbehandleNyeBarn,
     ): List<BehandlingBarn> {
         feilHvis(
             ustrukturertDokumentasjonType != UstrukturertDokumentasjonType.PAPIRSØKNAD &&
-                barnSomSkalFødes.isNotEmpty()
+                barnSomSkalFødes.isNotEmpty(),
```

Comment:
Dette blir ju også koko

Reply:
Enig, dette kan vel ikke være meningen fra ktlint sin side?

Reply:
Tolker det som at det er "ønskelig" for multiline-argument 👀 Gir vel kanskje litt mening, men ser stygt ut

Reply:
Skjønner. Ja, selv om det er best å ha mest mulig default, så ser regelen ut til å være en god kandidat til å ignorere